app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -363,21 +372,25 @@ def handle_successful_status_update(enrollment, response)
       reason: 'Successful status update',
       job_name: self.class.name,
     )
-    enrollment.profile.activate_after_passing_in_person
```

Comment:
This, and 373-380, are now conditional on the user not being `fraud_pending?`, so this got moved down. We still want the enrollment update below, though.